temoa/extensions/method_of_morris/morris_evaluate.py

```
@@ -30,7 +35,8 @@ def configure_worker_logger(log_queue, log_level):
     return worker_logger
 
 
-def evaluate(param_info, mm_sample, data, i, config: TemoaConfig, log_queue, log_level):
+def evaluate(param_info: dict[int, list[Any]], mm_sample: Any, data: dict[str, Any],
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `mm_sample`

(ANN401)